### ISSUE TITLE
2.10.4 installation / FATAL - Cant install CairoSVG

### ISSUE THREAD

**Issue by camillemonchicourt** [makinacorpus/Geotrek#1601]
I could install 2.10.4 a few weeks ago on my prod server. 
PnCevennes got an error when trying to install it yesterday. 
So I tried to install it on my test server and got the same error. 

```
Install Geotrek python dependencies...
Could not setup python environment !
```

``install.log`` ends with : 

```
TypeError: 'encoding' is an invalid keyword argument for this function
An error occurred when trying to install CairoSVG 2.0.0rc2. Look above this message for any errors that were output by easy_install.
While:
  Installing django.
  Getting distribution for 'CairoSVG>=0.4.1'.
Error: Couldn't install: CairoSVG 2.0.0rc2
make: *** [env_standalone] Erreur 1
+ success=2
+ '[' 2 -ne 0 ']'
+ exit_error 3 'Could not setup python environment !'
+ code=3
+ shift
+ echo_error 'Could not setup python environment !'
+ set +x
+ echo '(More details in install.log)'
(More details in install.log)
+ exit 3
```

@amandine-sahl could make it work by replacing 
```
eggs/WeasyPrint-0.23-py2.7.egg/EGG-INFO/requires.txt:5:CairoSVG==1.0.20
```
by 
```
eggs/WeasyPrint-0.23-py2.7.egg/EGG-INFO/requires.txt:5:CairoSVG>=0.4.1
```

But not sure it is a clean solution.

**Comment by gutard** [comment]
Know problem in Weasyprint : https://github.com/Kozea/WeasyPrint/pull/323
Just need to upgrade Weasyprint to version 0.29 that fixes it.

**Comment by gutard** [comment]
Anyway, cairosvg 2 seems removed from pypi. This currently fixes the problem.

**Comment by camillemonchicourt** [comment]
OK so I will try to install it again as it didnt worked yesterday. Thanks.

**Comment by camillemonchicourt** [comment]
OK I tried to install it again and got the same error : 

```  
File "/tmp/easy_install-04_Ds0/CairoSVG-2.0.0rc2/setup.py", line 23, in <module>
    'Pillow',
  File "/home/sentiers/Geotrek/lib/python2.7/site-packages/setuptools/sandbox.py", line 403, in _open
    return _open(path,mode,*args,**kw)
TypeError: 'encoding' is an invalid keyword argument for this function
An error occurred when trying to install CairoSVG 2.0.0rc2. Look above this message for any errors that were output by easy_install.
While:
  Installing django.
  Getting distribution for 'CairoSVG>=0.4.1'.
Error: Couldn't install: CairoSVG 2.0.0rc2
make: *** [env_standalone] Erreur 1
+ success=2
+ '[' 2 -ne 0 ']'
+ exit_error 3 'Could not setup python environment !'
+ code=3
+ shift
+ echo_error 'Could not setup python environment !'
+ set +x
+ echo '(More details in install.log)'
(More details in install.log)
+ exit 3
```

But yes, I edited ``conf/buildout.cfg`` with putting ``WeasyPrint = 0.29`` line 133 (https://github.com/makinacorpus/Geotrek/blob/f6fb2628ac91d1cca3c66d035227e7b1af6fb683/conf/buildout.cfg#L133) and launched the installation again and it worked. 

Thanks.

**Comment by gutard** [comment]
> OK I tried to install it again and got the same error

Did you tried on a fresh install ?

Currently, Pypi says latest version of cairosvg is 1.0.22, not 2.0.0rc2 (https://pypi.python.org/pypi/CairoSVG). So I'm not able to reproduce the problem. But actually, version 2.0.0rc2 do existst in pypi (https://pypi.python.org/pypi/CairoSVG/2.0.0rc2). So if you've already downloaded it you are stuck with it.

**Comment by camillemonchicourt** [comment]
I downloaded it again and installed from a new folder but maybe I made something wrong.
In anycase, it worked when I changed ``conf/buildout.cfg`` with ``WeasyPrint = 0.29``.

**Comment by submarcos** [comment]
fix in #1602

**Comment by camillemonchicourt** [comment]
Fixed in 2.11.0.
